classes/form/CustomerForm.php

```diff
@@ -117,15 +117,16 @@ public function validate()
 
         // birthday is from input type text..., so we need to convert to a valid date
         $birthdayField = $this->getField('birthday');
-        if (!empty($birthdayField->getValue()) &&
-            Validate::isBirthDate($birthdayField->getValue(), Context::getContext()->language->date_format_lite)
-        ) {
-            $dateBuilt = DateTime::createFromFormat(
-                Context::getContext()->language->date_format_lite,
-                $birthdayField->getValue()
-            );
-            $birthdayField->setValue($dateBuilt->format('Y-m-d'));
-        }
+        if (!empty($birthdayField) {
+          if (!empty($birthdayField->getValue()) &&
+              Validate::isBirthDate($birthdayField->getValue(), Context::getContext()->language->date_format_lite)
+          ) {
+              $dateBuilt = DateTime::createFromFormat(
+                  Context::getContext()->language->date_format_lite,
+                  $birthdayField->getValue()
+              );
+              $birthdayField->setValue($dateBuilt->format('Y-m-d'));
+         }}
```

Comment:
With this additionnal if, can you indent further this block in order to comply with [PSR-2](https://www.php-fig.org/psr/psr-2/#51-if-elseif-else) standard ? We're following it to get a more readable code 😉

Then the `}}` will become
```
    }
}
```

Reply:
The 4 indents are done